cg/apps/lims/api.py

```diff
@@ -128,35 +134,31 @@ def get_delivery_date(self, lims_id: str) -> dt.date:
             date = None
         return date
 
-    def capture_kit(self, lims_id: str) -> str:
+    def capture_kit(self, lims_id: str) -> str | None:
```

Comment:
```suggestion
    def get_capture_kit(self, lims_id: str) -> str | None:
```

Reply:
Same here, it's probably safer to address it in a separate PR and typehint all the calls